### PR TITLE
Fix runtime error when device is null

### LINKED ISSUE #904: Runtime error on Realtime page when device is null
## Summary

screen can be registered as null when through API (/api/collect).
But if device is null, the runtime page will be broken.

## Screenshot

Screen Shot 2022-01-03 at 13 18 00

### PR DESCRIPTION
**Pull request by p-chan**
Fix #904

### CODE REVIEW

File: components/metrics/RealtimeLog.js [hunk 1]
```diff
@@ -137,7 +137,11 @@ export default function RealtimeLog({ data, websites, websiteId }) {
             ),
             browser: <b>{BROWSERS[browser]}</b>,
             os: <b>{os}</b>,
-            device: <b>{intl.formatMessage(devices[device])?.toLowerCase()}</b>,
+            device: <b>
+                {devices[device]
+                  ? intl.formatMessage(devices[device]).toLowerCase()
+                  : intl.formatMessage({ id: 'label.unknown', defaultMessage: 'Unknown' })}
```
Comment (mikecao) [inline components/metrics/RealtimeLog.js:143]: You should pull the unknown message from `components/messages.js`.

**Review by mikecao — CHANGES_REQUESTED** [review]

File: components/metrics/RealtimeLog.js (hunk 1 above)
Comment (mikecao) [reply components/metrics/RealtimeLog.js:143]: Actually, there is a `getDeviceMessage` method in that file, use that instead.

Comment (p-chan) [reply components/metrics/RealtimeLog.js]: @mikecao Thank you for your review.
I've fixed it, so please review it again:pray:

File: components/metrics/RealtimeLog.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ export default function RealtimeLog({ data, websites, websiteId }) {
             ),
             browser: <b>{BROWSERS[browser]}</b>,
             os: <b>{os}</b>,
-            device: <b>{intl.formatMessage(devices[device])?.toLowerCase()}</b>,
+            device: <b>{getDeviceMessage(devices[device])}</b>,
```
Comment (mikecao) [inline components/metrics/RealtimeLog.js:140]: The method only takes `device` as input.

**Review by mikecao — CHANGES_REQUESTED** [review]

File: pages/api/collect.js [hunk 3]
```diff
@@ -9,6 +9,8 @@ import { createToken } from 'lib/crypto';
 export default async (req, res) => {
   await useCors(req, res);
 
+  console.log('hi!');
```
Comment (mikecao) [inline pages/api/collect.js:12]: Remove this.

**Comment by mikecao** [comment]
Will merge and fix.

**Pull request merged**